packages/sit-onyx/src/components/OnyxIconButton/OnyxIconButton.vue

```diff
@@ -0,0 +1,113 @@
+<script lang="ts" setup>
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import OnyxLoadingIndicator from "../OnyxLoadingIndicator/OnyxLoadingIndicator.vue";
+import type { OnyxIconButtonProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxIconButtonProps>(), {
+  disabled: false,
+  type: "button",
+  variation: "primary",
+  size: "24px",
+});
+
+defineSlots<{
+  /** Slot for an custom icon. Property `icon` will have no effect if custom content is passed. */
```

Comment:
```suggestion
  /** Slot for an custom icon. Property `icon` must be omitted. */
```

The v-if checks for the existence of the icon prop, not the slot

Reply:
true

Reply:
updated